.github/workflows/circuit-t0rn-collator-release.yml

```diff
@@ -0,0 +1,183 @@
+name: Circuit t0rn Rococo Collator Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+'
```

Comment:
will this work with the `-rc.x` tagging convention?

Reply:
yep, checked if GH triggers fir for both prod and candidates 👍